packages/fhir-care-team/src/components/CreateEditCareTeam/index.tsx

```diff
@@ -92,11 +100,19 @@ const CreateEditCareTeam: React.FC<CreateEditCareTeamProps> = (props: CreateEdit
 
   const buildInitialValues = getCareTeamFormFields(singleCareTeam.data as ICareTeam);
 
+  const disabledFields: string[] = [];
+  if (!hasReadOrgs) {
+    disabledFields.push(managingOrganizations);
+  }
+  if (!hasReadPractitioner) {
+    disabledFields.push(practitionerParticipants);
+  }
```

Comment:
would having the permission check defined statically on the formItem make things a bit more scalable? in the sense that re-using the form one need not recompute the disabled prop on that HOC component rendering the form.

Reply:
Actually for this I see the data fetchers are here as well. so I think it might be ok.